prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -204,7 +198,7 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
             settingCode: careSetting.careSettingCode,
             formControl: this.communityHealthEmails,
             subheaderContent: `Send your approval to your private community health practice\'s PharmaNet administrator (e.g., office
-              manager).`
+              manager). If you work in more than one clinic you can include the email addresses for all the PharmaNet administrators separated by a comma.`
```

Comment:
Since this text is the same for each care setting type, this could be put in the settingEmail template once instead.

Reply:
> Since this text is the same for each care setting type, this could be put in the settingEmail template once instead.

Mind pointing the location of settingEmail template? Also, this is just text on the page not the actual email sent.

Reply:
it is in the template file, `pharmanet-enrolment-summary.component.html` line 29. That is where this config here is being used. So since this part of the `subheaderContent` is the same for each care setting type, you can just put that sentence directly in the template after the `subheaderContent` on line 40/

Reply:
Got it. Fixed.👌